<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registWindow</class>
 <widget class="QWidget" name="registWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注 册 界 面</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用 户 名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密    码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="registButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注    册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registButton_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册并登录</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>registButton</sender>
   <signal>clicked()</signal>
   <receiver>registWindow</receiver>
   <slot>registClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>registButton_2</sender>
   <signal>clicked()</signal>
   <receiver>registWindow</receiver>
   <slot>loginClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>registClicked()</slot>
  <slot>loginClicked()</slot>
 </slots>
</ui>
